src/app/store/reducers/ReferenceData.reducers.spec.ts

```diff
@@ -32,14 +35,28 @@ describe('should update the ReferenceDataReducers with the correct state when th
     });
   });
 
+  describe('[LoadTestTypeCategoriesSuccess]', () => {
+    it('should update testTypeCategories value in state', () => {
```

Comment:
Same here we could add a better description to represent what is happening in the reducer which takes a state a dispatched action then return a new state
So we could have something like
"it should update the ReferenceDataReducers reducer state with the correct payload once LoadTestTypeCategoriesSuccess is dispatched"

Reply:
@alleypa suggested:
@irina005 Rephrase test titles for all test actions in this file. i,e. it should create LoadTestTypeCategoriesSuccess action. Action should create instances. Please update all